ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
And how would we see these 2 servers mentioned on the CS? As a comma-separated list of values in FTSEndpoint?
  FTSEndpoints
  {
    FTS3
    {
      FTSEndpoint = https://fts3-pilot.cern.ch:8446
      FTSType = FTS3
    }
  }